remme/tp/node_account.py

```diff
@@ -0,0 +1,103 @@
+import logging
+from sawtooth_sdk.processor.exceptions import InvalidTransaction
+
+from remme.protos.account_pb2 import AccountMethod
+from remme.protos.node_account_pb2 import (
+    NodeAccount, NodeAccountMethod, NodeAccountInternalTransferPayload, EmptyPayload
+)
+
+from remme.shared.forms import (
+    NodeAccountInternalTransferPayloadForm, EmptyPayloadForm
+)
+from remme.shared.constants import Events, EMIT_EVENT
+from remme.settings import SETTINGS_MINIMUM_STAKE
+
+from .basic import (
+    PB_CLASS, PROCESSOR, VALIDATOR, BasicHandler, get_data, get_multiple_data
+)
+from remme.settings.helper import _get_setting_value
+LOGGER = logging.getLogger(__name__)
+
+FAMILY_NAME = 'node_account'
+FAMILY_VERSIONS = ['0.1']
+
+
+class NodeAccountHandler(BasicHandler):
+    def __init__(self):
+        super().__init__(FAMILY_NAME, FAMILY_VERSIONS)
+
+    def get_state_processor(self):
+        return {
+            NodeAccountMethod.INITIALIZE_MASTERNODE: {
+                PB_CLASS: NodeAccountInternalTransferPayload,
+                PROCESSOR: self._initialize_masternode,
+                VALIDATOR: NodeAccountInternalTransferPayloadForm,
+            },
+            NodeAccountMethod.CLOSE_MASTERNODE: {
+                PROCESSOR: self._close_masternode,
+                PB_CLASS: EmptyPayload,
+                VALIDATOR: EmptyPayloadForm,
+            },
+            NodeAccountMethod.TRANSFER_FROM_UNFROZEN_TO_OPERATIONAL: {
+                PB_CLASS: NodeAccountInternalTransferPayload,
+                PROCESSOR: self._transfer_from_unfrozen_to_operational,
+                VALIDATOR: NodeAccountInternalTransferPayloadForm,
+            }
+        }
+
+    def _initialize_masternode(self, context, node_account_public_key, internal_transfer_payload):
```

Comment:
Remove init and close implementation from this pull request, please,

Reply:
Done. Everything related to init and close masternode was removed.